lib/omise/res/OmiseApiResource.php

```diff
@@ -250,7 +250,7 @@ private function _executeTest($url, $requestMethod, $key, $params = null)
      */
     private function genOptions($requestMethod, $userpwd, $params)
     {
-        $user_agent        = "OmisePHP/".OMISE_PHP_LIB_VERSION;
+        $user_agent        = "OmisePHP/" . OMISE_PHP_LIB_VERSION . " PHP/" . phpversion();
```

Comment:
@iwat nice catch 👍 yah, usually I have 1 space between string concat in other plugin projects. 
But better to not do it here for now, as you mentioned.

Will remove